libc/test/include/CMakeLists.txt

```diff
@@ -14,3 +14,22 @@ add_libc_test(
     # This is needed because the __containerof macro uses statement expression.
     -Wno-gnu-statement-expression-from-macro-expansion
 )
+
+add_libc_test(
+  stdbit_test
+  SUITE
+    libc_include_tests
+  SRCS
+    stdbit_test.cpp
+  DEPENDS
+    libc.__support.CPP.limits
+    libc.include.llvm-libc-macros.stdbit_macros
+    libc.src.stdbit.stdc_leading_zeros_uc
+    libc.src.stdbit.stdc_leading_zeros_ui
+    libc.src.stdbit.stdc_leading_zeros_ul
+    libc.src.stdbit.stdc_leading_zeros_ull
+    libc.src.stdbit.stdc_leading_zeros_us
+  COMPILE_OPTIONS
+    # stdbit is full of type generic macros implemented via C11 _Generic.
```

Comment:
if you are using C11 features then why is the flag `-Wno-c11-extensions`? This comment may need some more detail.

Reply:
added more info in dfbf115e640e, but let me know if there's a better way to word this.